augur/application/cli/backend.py

```diff
@@ -107,8 +107,8 @@ def start(disable_collection, development, port):
             os.remove("celerybeat-schedule.db")
 
         scheduling_worker = f"celery -A augur.tasks.init.celery_app.celery_app worker -l info --concurrency=1 -n scheduling:{uuid.uuid4().hex}@%h -Q scheduling"
-        core_worker = f"celery -A augur.tasks.init.celery_app.celery_app worker -l info --concurrency=14 -n core:{uuid.uuid4().hex}@%h"
-        secondary_worker = f"celery -A augur.tasks.init.celery_app.celery_app worker -l info --concurrency=5 -n secondary:{uuid.uuid4().hex}@%h -Q secondary"
+        core_worker = f"celery -A augur.tasks.init.celery_app.celery_app worker -l info --concurrency=70 -n core:{uuid.uuid4().hex}@%h"
```

Comment:
We are scheduling 80 repos for 70 processes (60 on primary 20 on facade). I don't think we should do this. I think we _really_ should schedule a number of repos less than or equal to the number of processes we have. Otherwise we put things in rabbitmq when we don't have to. I think we shouldn't use rabbitmq if we don't have to bc large queue sizes have been an issue in the past.

Reply:
We should not schedule more repos than processes because it provides no benefit and can only cause issues, as we already have the information for that pending repo in the db. If we schedule more repos than processes we will have a repo in the rabbitmq queue doing nothing because it has no processes. Repos that are running still _have_ to use rabbitmq because there are multiple jobs per repo and it has to store the messages for these jobs in rabbitmq. And repos that don't have available processes should not have messages in rabbitmq